talk/owt/sdk/include/cpp/owt/base/stream.h

```diff
@@ -158,6 +172,69 @@ class Stream {
   mutable std::mutex observer_mutex_;
   std::vector<std::reference_wrapper<StreamObserver>> observers_;
 };
+
+#ifdef OWT_ENABLE_QUIC
+/// A QuicStream can be fetched from a published LocalStream for data,
+/// on which you can write to MCU;
+/// Or from a subscription from MCU for data, on which you can read.
+class QuicStream : public owt::quic::QuicTransportStreamInterface::Visitor {
```

Comment:
I'm wondering if we can just use owt::quic::QuicTransportStreamInterface here? Wrapping an SDK object with another namespace seems to be redundant.

Reply:
Plan is to hide the Quic SDK interface from application, so further they will not need to include the QUIC SDK headers for building their application. Although this is not yet complete, this would be our target.